.travis.yml

```diff
@@ -42,11 +52,20 @@ matrix:
       install: *_install
       after_success:
         - codecov
+    - stage: deploy
+      python: 3.4
+      install:
+        - pip3 install --upgrade pip
+        - pip3 install twine
+      before_script: skip
+      script:
+        - python3 setup.py bdist_wheel
+      deploy: *_deploy
```

Comment:
Don't offend Python 3.5 and 3.6

Reply:
But we have `sourced_ml-0.4.4-py3-none-any.whl` as output, so it should work for any python3. And that the reason why we have only one wheel for python3.4,5,6 and need to use stages (your second comment.).

Correct me, if I am wrong.

Reply:
Right, there are no native extensions. Regarding stages, you can restrict deployment to specific python version, so stages are not required.

Reply:
but we first need to test our code for all pythons and only after that deploy. 

if we restrict deployment to specific python version, we can publish wheel that will fail on another pythons.

Reply:
Fair enough again since we do not build binary packages. My desperate attempt to speed up CI failed